Sources/POSIX/gmtime.swift

```diff
@@ -0,0 +1,44 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2015 - 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import typealias libc.time_t
+import func libc.strftime
+import func libc.gmtime
+import func libc.time
+
+public enum StringError: ErrorProtocol {
+    case NotEnoughSpace
+}
+
+extension StringError: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .NotEnoughSpace:
+            return "gmtime error: not enough space, increase resultSize."
+        }
+    }
+}
+
+public func gmtime(format: String, resultSize: Int = 200) throws -> String {
```

Comment:
I've split `gmtime` and `strftime`.  Now gmtime returns `tm` structure.
And `strftime` does standard posix string formatting :)